blog.html

```diff
@@ -22,6 +22,286 @@
+                        See <a href="https://kafka.apache.org/40/documentation.html#consumer_rebalance_protocol">here</a> for details.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/B40ODg">KIP-890: Transactions Server-Side Defense</a>
+                        <br>
+                        The second phase of KIP-890 has been completed. It reduces the chances of “zombie transactions”
+                        during producer failures. See <a href="https://kafka.apache.org/40/documentation.html#transaction_protocol">here</a> for details.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/4hA0Dw">KIP-932: Queues for Kafka (Early Access)</a>
+                        <br>
+                        This KIP introduces the concept of a share group as a way of enabling cooperative consumption
+                        using Kafka topics. It does not add the concept of a “queue” to Kafka per se, but rather that
+                        introduces cooperative consumption to accommodate these queuing use-cases using regular Kafka
+                        topics. Share groups make this possible. You can think of a share group as roughly equivalent to a
+                        “durable shared subscription” in existing systems.
+                        Please see the <a href="https://cwiki.apache.org/confluence/x/i4ogF">Early Access Release Notes</a>
+                        for limitations and configurations.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/mpOzDw">KIP-966: Eligible Leader Replicas (Preview)</a>
+                        <br>
+                        KIP-966 introduces Eligible Leader Replicas (ELR) in preview in 4.0. ELR is a subset of the ISR
+                        replicas guaranteed to have complete data up to the high-watermark. ELRs are safe for leader
+                        election, preventing data loss. See <a href="https://kafka.apache.org/40/documentation.html#eligible_leader_replicas">here</a> for details.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/MYtEE">KIP-996: Pre-Vote</a>
+                        <br>
+                        KIP-996 introduces a "Pre-Vote" mechanism to reduce unnecessary KRaft leader elections in Apache
+                        Kafka clusters. By allowing nodes to check their eligibility for leadership before initiating an election,
+                        this approach minimizes disruptions caused by network partitions or transient issues.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/joqzDw">KIP-950: Tiered Storage Disablement</a>
```

Comment:
This has already been shipped in 3.9 (and added in it's blog post). We can remove it from here.

Reply:
Ack. I will remove it.